sideboard/websockets.py

```diff
@@ -376,6 +406,9 @@ def __init__(self, *args, **kwargs):
             meaningful) and /wsrpc being client-cert protected (so the username
             will always be 'rpc').
 
+        header_fields: We copy header fields from the request that initiated the
+            websocket connection.
```

Comment:
MAGFest doesn't currently care about this, but in the future we might if we start doing more with exposed RPC services over websocket.